lib/routes/account.js

```diff
@@ -579,7 +599,8 @@ module.exports = function (
                           // value; Its stable, random, and will change on
                           // account reset.
                           wrapKb: account.wrapWrapKb,
-                          emailVerified: true
+                          emailVerified: true,
+                          tokenVerificationId: tokenVerificationId
```

Comment:
We did, and it isn't - just want to make sure that's what's intended here.

(I guess it's possible that they'll get verified in the future, if you try to use one for sync and it says you're unverified and then you use the resend_email link to send a verification link....so this all seems OK to me)